code/src/style.css

```diff
@@ -0,0 +1,219 @@
+* {
```

Comment:
I really love your styling, Caroline! Minimalistic yet stylish, the colours are beautiful ✨ and the input boxes are easy to read from. 😍 Big plus for making the app optimized for other devices.